src/pybind/mgr/dashboard/frontend/src/app/shared/services/auth-interceptor.service.ts

```diff
@@ -1,43 +1,50 @@
 import {
-  HttpErrorResponse, HttpEvent, HttpHandler, HttpInterceptor, HttpRequest
+  HttpErrorResponse,
+  HttpEvent,
+  HttpHandler,
+  HttpInterceptor,
+  HttpRequest
 } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { Router } from '@angular/router';
 
-import { ToastsManager } from 'ng2-toastr';
 import 'rxjs/add/observable/throw';
 import 'rxjs/add/operator/catch';
 import { Observable } from 'rxjs/Observable';
 
+import { NotificationType } from '../enum/notification-type.enum';
 import { AuthStorageService } from './auth-storage.service';
+import { NotificationService } from './notification.service';
 
 @Injectable()
 export class AuthInterceptorService implements HttpInterceptor {
-
-  constructor(private router: Router,
-              private authStorageService: AuthStorageService,
-              public toastr: ToastsManager) {
-  }
+  constructor(
+    private router: Router,
+    private authStorageService: AuthStorageService,
+    public notif: NotificationService
```

Comment:
Please rename `notif` to something more meaningful.

Reply:
Done